lte/gateway/c/core/oai/tasks/s6a/s6a_grpc_iface.hpp

```diff
@@ -18,7 +18,7 @@
 #ifndef S6A_GRPC_IFACE_H_SEEN
 #define S6A_GRPC_IFACE_H_SEEN
```

Comment:
Replace this with #pragma once

Reply:
I found some statement on wiki which is not very supportive for use of #pragma once:
https://en.wikipedia.org/wiki/Pragma_once#:~:text=Caveats%5Bedit,pragma%20once.

@ssanadhya What is your opinion on this?

Reply:
@VinashakAnkitAman , the caveats highlighted on the Wiki (such as symbolic links, networked file systems, etc.) do not apply to Magma codebase. So let's continue using the `#pragma once` directive, since it makes it easier for contributors.

Reply:
Sure @ssanadhya.. Thanks for the clarification. I will update the PR.